Source/Scene/Cesium3DTileStyle.js

```diff
@@ -294,9 +312,62 @@ define([
             set : function(value) {
                 this._meta = value;
             }
-        },
-
+        }
     });
 
+    /**
+     * Gets the color shader function for this style.
+     *
+     * @param {String} functionName Name to give to the generated function.
+     * @param {String} attributePrefix Prefix that is added to any variable names to access vertex attributes.
+     * @param {Object} shaderState Stores information about the generated shader function, including whether it is translucent.
+     *
+     * @returns {String} The shader function.
+     *
+     * @private
+     */
+    Cesium3DTileStyle.prototype.getColorShaderFunction = function(functionName, attributePrefix, shaderState) {
+        if (this._colorShaderFunctionReady) {
+            // Return the cached result, may be undefined
+            return this._colorShaderFunction;
+        }
+
+        this._colorShaderFunctionReady = true;
+        this._colorShaderFunction = this.color.getShaderFunction(functionName, attributePrefix, shaderState, 'vec4');
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(this._colorShaderFunction)) {
+            throw new DeveloperError('Could not generate valid shader code for the color style.');
```

Comment:
It shouldn't be hard to give more detailed error messages for this inside `Node.getShaderExpression`.